scripts/scripts.js

```diff
@@ -236,6 +236,9 @@ async function loadEager(doc) {
     shoppingCartContext: {
       totalQuantity: 0,
     },
+    shopperContext: {
+      shopperId: getUserTokenCookie() ? 'logged-in' : 'guest',
```

Comment:
@herzog31 do you recall why we set some context here vs in delayed? Nothing is consumed until delayed.js loads and subsequently the collector, but these are the first context values pushed to the datalayer on page load (as they are in eager).

Reply:
Of course -.-